docs/source/common/trainer.rst

```diff
@@ -1544,8 +1544,8 @@ val_check_interval
 How often within one training epoch to check the validation set.
 Can specify as float or int.
 
-- use (float) to check within a training epoch
-- use (int) to check every n steps (batches)
+- pass a ``float`` to check after a percentage of the training epoch
```

Comment:
I would not mention percentage because that's a value outside the range 0-1. Should we say "fraction" instead?
```suggestion
- pass a ``float`` in the range [0.0, 1.0] to check after a fraction of the training epoch
```
something like that?